iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -103,9 +103,9 @@ public static Collection inputs() throws Exception
 
     // Max time to wait to see it on Hub
     protected static final long PERIODIC_WAIT_TIME_FOR_VERIFICATION = 1000; // 1 sec
-    protected static final long MAX_WAIT_TIME_FOR_VERIFICATION = 3000; // 30 sec
```

Comment:
This was labeled as 30 seconds, but it was actually only 3 seconds

Reply:
And I assume 3 seconds wasn't long enough and extending the wait period will fix the flakiness?

Reply:
Yeah